src/Compilers/Core/Portable/InternalUtilities/EncodingExtensions.cs

```diff
@@ -56,5 +56,20 @@ internal static bool TryGetMaxCharCount(this Encoding encoding, long length, out
 
             return false;
         }
+
+#if !NETCOREAPP
+        internal static unsafe string GetString(this Encoding encoding, ReadOnlySpan<byte> buffer)
+        {
+            if (buffer.Length == 0)
+            {
+                return string.Empty;
+            }
+
+            fixed (byte* bytes = buffer)
```

Comment:
The BCL implementation of the `GetString(ReadOnlySpan<byte>)` defers to the `GetString(byte*)` overload in the same way that I'm doing here.